api/src/main/java/org/openmrs/api/db/hibernate/HibernatePatientDAO.java

```diff
@@ -249,16 +249,17 @@ public void deletePatientIdentifierType(PatientIdentifierType patientIdentifierT
 		// join with the patient table to prevent patient identifiers from patients
 		// that already voided getting returned
 		criteria.createAlias("patient", "patient");
+
+		// make sure the patient isn't voided
```

Comment:
@vixus0 can you please remove this comment. the code after that says the same thing